src/base/dimensionmanager.js

```diff
@@ -48,7 +56,7 @@ const DimensionManagerPrototype = {
           // no more entities models left
           modelName = "entities_" + dim;
           const newEntities = {
-            [modelName]: Object.assign(Entities.getClassDefaults(), { dim })
+            [modelName]: { dim }
```

Comment:
why not to use Entities.getClassDefaults()? when constructing entities object?
aren't we going to loose `show: {}` and `skipFilter` from the model then?
also if we add more defaults this code would not get them

Reply:
looks like nothing loose
initSubmodel->new Modl(Entities)->init->initialValues=deepExtend({}, this.getClassDefaults(),values)->set(initialValues)

Reply:
Ah, you're saying defaults are already added later in constructing Entities model. Yeap, okay!